WPMedia/engine-theme-sdk
# PEN-1392 add social image metadata

## Description
Updated MetaData component to add fallback site image as social image metadata. In the case the Authors page type, the image will be the author photo if exists.

## Jira Ticket
- [PEN-1392](https://arcpublishing.atlassian.net/browse/PEN-1392)

## Acceptance Criteria
As per this ALC document, image metadata should display for all page types, if it is present. It should be the Fallback Image set for the website. (If the fallback image is not present, then no image should display). https://corecomponents.arcpublishing.com/alc/arc-products/themes/user-docs/page-template-guide/page-types/

The image should appear in both twitter:image and og:image meta tags

## Test Steps
- check on `/pf/tags/beaches` the image and alt:image used is the fallback one
- check on `/pf/section/news` the image and alt:image used is the fallback one
- check on `/pf/author/alexandra-petri` the image and alt:image used the one from the author
- check on `/pf/homepage` the image and alt:image used is the fallback one

## Effect Of Changes

### After
<img width="906" alt=" 2020-11-04-15 37 11" src="https://user-images.githubusercontent.com/9757/98162232-d0775380-1ebf-11eb-82c7-22991d5129b1.png">

## Dependencies or Side Effects
- 

## Review Checklist
please explain why (so that we can fix it whenever it gets refactored).


## NOTE

I bypassed the pre-commit hook because errors on the chromatic component. I executed `jest`and `lint` and all green before generate this PR.

<img width="975" alt="Screen Shot 2020-11-04 at 15 53 36" src="https://user-images.githubusercontent.com/9757/98163116-1da7f500-1ec1-11eb-93fa-118ebfff0298.png">
